gravitee-apim-cypress/cypress/model/apis.ts

```diff
@@ -67,6 +67,30 @@ export interface PortalApi {
   categories: string[];
 }
 
+export enum ApiPlanType {
+  API = 'API',
+  CATALOG = 'CATALOG',
+}
+
+export enum ApiPlanSecurityType {
+  KEY_LESS = 'KEY_LESS',
+  API_KEY = 'API_KEY',
+  OAUTH2 = 'OAUTH2',
+  JWT = 'JWT',
+}
+
+export enum ApiPlanValidationType {
+  AUTO = 'AUTO',
+  MANUAL = 'MANUAL',
+}
+
+export enum ApiPlanStatus {
+  STAGING = 'STAGING',
+  PUBLISHED = 'PUBLISHED',
+  CLOSED = 'CLOSED',
+  DEPRECATED = 'DEPRECATED',
+}
```

Comment:
The interface definitions ApiPlanType, ApiPlanSecurityType, ApiPlanValidationType & ApiPlanStatus already exist in model/plan.ts (same name just without the prefix 'ApiPlan')

Reply:
Thanks, that has been refactored in this PR https://github.com/gravitee-io/gravitee-api-management/pull/1071